libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshController.kt

```diff
@@ -34,21 +37,46 @@ internal class RouteRefreshController(
     private val localDateProvider: () -> Date
 ) {
 
+    @Volatile
```

Comment:
```suggestion
```
There is only one thread, is volatile required?

Reply:
after removed `fun onCancel` it's not needed anymore 👍